src/Main.cs

```diff
@@ -59,14 +65,21 @@ private void OnGUI() {
 			GUILayout.EndHorizontal();
 
 			randomizeSpawns.Value = GUILayout.Toggle(randomizeSpawns.Value, "Randomize Spawns");
-
+			
 			if (currentPlayers == null || currentPlayers.Count == 0) {
 				GUILayout.EndVertical();
 				GUILayout.EndArea();
 				return;
 			}
-
+			
+			if (GUILayout.Button("Ready all"))
+				currentPlayers.ForEach(player => SetReadyState(player, true));
+			
+			if (GUILayout.Button("Not Ready all"))
+				currentPlayers.ForEach(player => SetReadyState(player, false));
```

Comment:
Braces should be covered by clang-format, please run it on your code
```suggestion
			if (GUILayout.Button("Ready All")) {
				currentPlayers.ForEach(player => SetReadyState(player, true));
			}
			if (GUILayout.Button("Un-Ready All")) {
				currentPlayers.ForEach(player => SetReadyState(player, false));
			}
```